pkg/mkimage-raw-efi/make-raw

```diff
@@ -238,17 +252,27 @@ do_usb_conf() {
 }
 
 #
-# Extract partitions from stdin
+# Extract partitions from stdin or look them up in /bits
 #
-mkdir /parts 2>/dev/null || :
-[ -t 0 ] || (cd /parts; bsdtar xzf -)
+if [ ! -d /parts -o "$(cd /parts 2>/dev/null ; echo *)" = '*' ]; then
+   mkdir /parts
+   cd /parts
+   [ "$(cd /bits ; echo *)" = '*' ] || ln -s /bits/* .
+fi
+
+cd /parts
+[ -t 0 ] || bsdtar xzf -
```

Comment:
Is this-t 0 check still a good idea? Might break if the build is run differently.
Can't you put the bsdtar in an else branch at L261 instead?

Reply:
ok